payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -296,6 +304,25 @@ class GooglePayPaymentMethodLauncher @AssistedInject internal constructor(
             get() = merchantCountryCode.equals(Locale.JAPAN.country, ignoreCase = true)
     }
 
+    @Parcelize
+    data class ShippingAddressConfig @JvmOverloads constructor(
+        /**
+         * Set to true to request a full shipping address.
```

Comment:
What does this mean? Setting it to `true` will make sure that `Result.Completed` contains a value on `shippingInformation`?

Reply:
Setting it to true will display the shipping addresses in the Google Pay sheet. It will also return a shipping address in the result. Could make the comments better here.